lib/suse/connect/templates/extensions_list.text.erb

```diff
@@ -1,36 +1,23 @@
 \e[1mAVAILABLE EXTENSIONS AND MODULES\e[0m
 
-<% if grouped_extensions[true] %>
-\e[1mFREE EXTENSIONS\e[0m
-
-<% grouped_extensions[true].each do |extension| -%>
-  \e[1m<%= extension[:name] %>\e[0m
-  Install with: SUSEConnect -p <%= extension[:activation_code] %>
-<% extension[:extensions].each do |additional_extension| -%>
-    \e[1m<%= additional_extension[:name] %>\e[0m
-    Install with: SUSEConnect -p <%= additional_extension[:activation_code] %>
-
-  <% end -%>
-
-<% end -%>
-<% end -%>
-
-<% if grouped_extensions[false] %>
-\e[1mPAID EXTENSIONS\e[0m
-
-<% grouped_extensions[false].each do |extension| -%>
-  \e[1m<%= extension[:name] %>\e[0m
-  Install with: SUSEConnect -p <%= extension[:activation_code] %> -r \e[32m\e[1mREGCODE\e[0m
-<% extension[:extensions].each do |additional_extension| -%>
-
-    \e[1m<%= additional_extension[:name] %>\e[0m
-    Install with: SUSEConnect -p <%= additional_extension[:activation_code] %> -r \e[32m\e[1mREGCODE\e[0m
-  <% end -%>
-
-<% end -%>
-<% end -%>
-
+<%
+any_installed = installed_products.select {|x| !x.isbase }.any?
+{ true => 'FREE EXTENSIONS', false => 'PAID EXTENSIONS' }.each do |type_key, title| %>
+\e[1m<%= title %>\e[0m
+
+<%
+  grouped_extensions[type_key].each do |extension| -%>
```

Comment:
That throws an exception when running `SUSEConnect --list-extensions` on a system registered against SMT when `type_key` = `false`.
(It seems there are no paid extensions available specifically for my system)

Here's the trace:
```
SUSEConnect error: NoMethodError: undefined method `each' for nil:NilClass
(erb):9:in `block in render'
(erb):5:in `each'
(erb):5:in `render'
/usr/lib64/ruby/2.1.0/erb.rb:847:in `eval'
/usr/lib64/ruby/2.1.0/erb.rb:847:in `block in result'
/usr/lib64/ruby/2.1.0/erb.rb:848:in `call'
/usr/lib64/ruby/2.1.0/erb.rb:848:in `result'
/root/connect/lib/suse/toolkit/renderer.rb:13:in `render'
/root/connect/lib/suse/connect/status.rb:39:in `print_extensions_list'
/root/connect/lib/suse/connect/cli.rb:31:in `execute!'
./SUSEConnect:11:in `<main>'
```